lib/http/form_data.rb

```diff
@@ -57,10 +57,9 @@ def create(data, encoder: nil)
       # @raise [Error] `obj` can't be coerced.
       # @return [Hash]
       def ensure_hash(obj)
-        case
-        when obj.nil?               then {}
-        when obj.is_a?(Hash)        then obj
-        when obj.respond_to?(:to_h) then obj.to_h
+        if    obj.nil?               then {}
+        elsif obj.is_a?(Hash)        then obj
+        elsif obj.respond_to?(:to_h) then obj.to_h
```

Comment:
This doesn't look prettier. I wonder what's the reason of this change?

Reply:
Rubocop :D

Reply:
I guess it doesn't look much worse either... :D